kafkacat.c

```diff
@@ -936,6 +942,7 @@ static void RD_NORETURN usage (const char *argv0, int exitcode,
                 "  -p -1              Use random partitioner\n"
                 "  -D <delim>         Delimiter to split input into messages\n"
                 "  -K <delim>         Delimiter to split input key and message\n"
+                "  -F <str>           Use a fixed key for all messages\n"
```

Comment:
I think `-k` would be more intuitive

Reply:
Agreed. Funny I didn't think of that alternative.